src/unit_tests/mocks/styleMock.js

```diff
@@ -1,3 +0,0 @@
-module.exports = {
```

Comment:
Using [styleMock with idObj proxy](https://github.com/onfido/onfido-sdk-ui/blob/improvement/convert-App-and-redux-to-ts/src/unit_tests/mocks/styleMock.js) is better in this case, because we can render and assert correct class names with `enzyme`